<a href="https://colab.research.google.com/github/bindumadhavinadikuppala/Basic--ML/blob/main/My%20first%20notebook%20in%20colab.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

                EVEN OR ODD

In [ ]:
n=int(input())
if (n%2==0):
    print("even")
else:
    print("odd")

14
even


 Explanation:
 
1)This is the program which determines whether the given number is even or odd
 
2)To find whether the number is even or odd an input must be considered 
 
3)Here n is considered as an input.Since the number is under integer category the keyword int is used
 
4)The condition for a number to become even is when it is divided by 2 the remainder must be 0 or the number will be a odd number
 
5)Here if-else condition is considered
 
6)Also here to we use the keyword print to get the desired output
 
7)14 is given as input and has been executed which gave an output even

         FACTORIAL OF A GIVEN NUMBER

In [16]:
n=int(input())
fact=1
for i in range(1,n+1):
    fact=fact*i
print(fact)

7
5040


 Explanation:
 
1)To find the factorial of a given number the above program is used
 
2)To find the factorial an input must be read,here n is considered as an input.Since it is integer keyword int is used
 
3)We have set fact value as 1 so 1 is stored in fact
 
4)In the program we used for loop and had taken the range of i from 1 to n+1
 
5)Now each value of i is multiplied with fact and stores the value
 
6)The keyword print is used to print the factorial of given number once the loop is ended
 
7)7 is given as input for the above program and had been executed which gave an output as 5040

          SUM OF FIRST n NATURAL NUMBERS

In [17]:
n=int(input())
sum=0
for i in range (1,n+1):
   sum=sum+i
   i=i+1
print(sum)

9
45


Explanation:
 
1)To find the sum of first n natural numbers the above program is used
 
2)A input n is read,since an integer is to be read keyword int is considered
 
3)We have set the value of sum as 0
 
4)In the program for loop is used and the range of i is taken from 1 to n+1
 
5)Now each value of i is added to sum and the values are stored
 
6)The keyword print is used to print the sum of first n natural numbers once the loop is ended
 
7)9 is given as input and the program is executed which gave an output of 45